### ISSUE TITLE
The Similar column is the ground truth from you or the result from code

### ISSUE THREAD

**Issue by kaitorecca** [adumrewal/SIFTImageSimilarity#6]
Thanks for your great work. I have question related to similar column in your readme.md, is has yes/no but cannot find the code in your repo related to this classification. So I just want to ask whether it's groundtruth from you?

**Comment by adumrewal** [comment]
You are right. The cut-off is not mentioned in the code. Based on experiments, following were the cut-offs:
- `> 1.0` : match
- `> 5.0` : 100% confidence match

This score is the one returned by `calculateScore` function in the jupyter notebook.

I hope this helps.
